.github/workflows/ci.yml

```diff
@@ -82,30 +85,25 @@ jobs:
           - windows-latest
 
         php-version:
-          - "7.4"
+          - "8.3"
 
         php-ini-values:
           - assert.exception=1, zend.assertions=1
 
         dependencies:
-          - locked
+          - highest
 
         include:
           - os: ubuntu-latest
             php-version: "7.4"
             dependencies: lowest
 
           - os: ubuntu-latest
-            php-version: "7.4"
+            php-version: "8.3"
             dependencies: highest
 
           - os: ubuntu-latest
-            php-version: "8.0"
-            dependencies: highest
-            php-ini-values: assert.exception=1, zend.assertions=1, opcache.enable=1, opcache.enable_cli=1, opcache.optimization_level=-1, opcache.jit_buffer_size=4096M, opcache.jit=1205
-
-          - os: ubuntu-latest
-            php-version: "8.1"
+            php-version: "8.3"
```

Comment:
we have two 8.3 and highest here - do we need both? is it because of the ini values?

Reply:
The tests were previously 8.0 and 8.1. As you said, there is no need to do 8.3 twice. The base test should be 8.2, so if we wanted to test all versions, we could list 8.0, 8.1 and 8.3 here. Maybe it's okay to not do all of versions and just pick some representative ones.